R/mod_read_spreadsheet.R

```diff
@@ -12,7 +12,7 @@
 #'
 #' @keywords internal
 #' @export 
-#' @importFrom shiny NS tagList 
+#' @importFrom shiny NS tagList googlesheets4
```

Comment:
Why is this needed?

Reply:
I think it is for documentation, to tell what dependencies the function has

Reply:
please accept or reject before we merge.